lib/cdo/aws/s3_packaging.rb

```diff
@@ -54,7 +54,7 @@ def update_from_s3
   # Uploads the created package to s3
   # @return package
   def upload_package_to_s3(package)
-    raise "Generated different package for same contents" unless package_matches_download(package)
+    @logger.warn "Different package found on S3 for the same contents" unless package_matches_download(package)
```

Comment:
if you are going to be changing this string, how about including `@package_name` in the output, i.e. `"Different #{@package_name} package..."`